### PR TITLE
💅 fresh coat of paint

### LINKED ISSUE #1: Add links to page template
Google crawled through and found the individual pages to the projects, which lacks a link to the code source. 

This is fine if someone is viewing the entire home page at [stodevx.github.io](https://stodevx.github.io) but not so good if they are routed to an actual project page, such as [stodevx.github.io/all-about-olaf.html](http://stodevx.github.io/all-about-olaf.html)

### PR DESCRIPTION
**Pull request by drewvolz**
Closes #1.

Big takeaway: in addition to a clean new look, this uses the github API to populate data.

This PR uses the base template from github's personal website [template](https://github.com/github/personal-website).
 
Tweaked a few things to get it to fit our org
* set the `repository` key in the config to "stodevx"
  * (if you build locally it looks at the remote source, so check what's at `git remote -v`)
* use `description` instead of `bio` in the meta description and the masthead
* deleted the "for hire" check and rendered html because it'll always be false
* removed the social media links from the commented-out `social_media` links from the config

<hr>

There's room to grow! 
* If anyone feels so inclined to write a blog post, it'll show up underneath the repositories (feel free to move that section to the _top_ of the page so we can show off your thoughts). Read more about creating a blog post in the README.
* I would like to tweak the commented-out "interests" section to be a github-sourced repository list of projects we are fans of rather than the content it supports now.

<details>
<summary>📸 Old</summary>

stodevx old

</details>

<details>
<summary>📸 New</summary>

stodevx new

</details>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 _site
 .sass-cache
 .jekyll-metadata
+
+# OSX
+.DS_Store
```
Comment (rye) [inline .gitignore]: I don't know if I want this to be kept here—I think people should ignore these themselves with their git client.

File: Gemfile [hunk 2]
```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
-
 gem 'github-pages', group: :jekyll_plugins
+gem "jekyll-github-metadata"
```
Comment (rye) [inline Gemfile]: Single quotes please.

File: LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2019 GitHub
```
Comment (rye) [inline LICENSE.txt]: GitHub doesn't own this theme? Or do they. Seems like we're giving them copyright over our site.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
+# Get started building your personal website
```
Comment (rye) [inline README.md]: Blitz this file please.

Comment (drewvolz) [reply README.md]: I suppose I could rework the readme to provide more relevant sections. I was thinking instructions on how to create a blog entry or modify content would still be useful, or how to run this repo on your own machine?

File: .gitignore (hunk 1 above)
Comment (drewvolz) [reply .gitignore]: If you feel strongly, I can get rid of it and just request changes if it ever becomes an issue.

File: README.md (hunk 4 above)
Comment (drewvolz) [reply README.md]: Blitzed.

**Comment by drewvolz** [comment]
I'm going to go ahead and merge this in.

**Pull request merged**